test/unit/org/apache/cassandra/db/compaction/unified/ControllerTest.java

```diff
@@ -121,6 +121,50 @@ public void testValidateOptionsIntegers()
         testValidateOptions(true);
     }
 
+    @Test
+    public void testValidateOptionsInvalidTargetSSTableSize()
```

Comment:
I strongly recommend `org.assertj.core.api.Assertions#assertThatThrownBy(org.assertj.core.api.ThrowableAssert.ThrowingCallable)` for the failure handling.  If the errors don't match what you expect then it produces a nicer error.

If you want, you can hand roll, that is fine, but you must actually validate the response; `catch () {}` isn't allowed in the code base

Reply:
I have made the changes. Thanks